bucket/vcpkg.json

```diff
@@ -0,0 +1,42 @@
+{
+    "version": "2020.11-1",
+    "description": "C++ Library Manager for Windows, Linux, and MacOS.",
+    "homepage": "http://aka.ms/vcpkg",
+    "license": "MIT",
+    "notes": [
+        "vcpkg is installed with telemetry disabled flag (-disableMetrics) for privacy sake.",
+        "In order to use vcpkg with Visual Studio, run the following command (may require administrator elevation):",
+        "`vcpkg integrate install`",
+        "The best way to use installed libraries with cmake is via the toolchain file scripts\\buildsystems\\vcpkg.cmake.",
+        "To use this file, you simply need to add it onto your CMake command line as:",
+        "`-DCMAKE_TOOLCHAIN_FILE=D:\\src\\vcpkg\\scripts\\buildsystems\\vcpkg.cmake`",
+        "Packages will be installed on $env:USERPROFILE\\.vcpkg\\."
```

Comment:
Is `$env:USERPROFILE` the best place to put vcpkg packages? Scoop has the _persist_ directory which might be better suited?